iroh-dns-server/src/metrics.rs

```diff
@@ -9,7 +9,7 @@ use struct_iterable::Iterable;
 pub struct Metrics {
     pub pkarr_publish_update: Counter,
     pub pkarr_publish_noop: Counter,
-    pub pkarr_publish_error: Counter,
+    // pub pkarr_publish_error: Counter,
```

Comment:
this is obviously guessing at usage.  but i'm surprised you want to remove this.  i'm used to seeing most counters in at least a pair of `_total` and `_error` so you can do error ratios.

Reply:
I agree, however these are unused counters and I think those can be misleading as you expect to see something in errors yet they never increment even if there was an error.